scripts/build_summary/Dockerfile

```diff
@@ -1,9 +1,28 @@
 FROM ubuntu:16.04
-RUN apt-get update && apt-get install -y groovy2 python3-pip build-essential python3-dev libssl-dev curl libffi-dev sudo git-core python3-lxml python-dev python-pip
+RUN apt-get update \
+ && apt-get install -y \
+      groovy2 \
+      python3-pip \
+      build-essential \
+      python3-dev \
+      libssl-dev \
+      curl \
+      libffi-dev \
+      sudo \
+      git-core \
+      python3-lxml \
+      python-dev \
+      python-pip \
+ && rm -rf /var/lib/apt/lists/*
```

Comment:
Perhaps using ``apt-get clean`` would also be useful here if we're intending to do some cleanup to maintain a clean image?

Reply:
According to https://docs.docker.com/develop/develop-images/dockerfile_best-practices/ this should already be done.